python/ray/serve/_private/logging_utils.py

```diff
@@ -51,12 +51,18 @@ def configure_component_logger(
             ray._private.worker._global_node.get_logs_dir_path(), "serve"
         )
         os.makedirs(logs_dir, exist_ok=True)
+        max_bytes = ray._private.worker._global_node.max_bytes
+        backup_count = ray._private.worker._global_node.backup_count
```

Comment:
nit: could you add these as parameters to this function and if they aren't provided pick them up from the ray variables?

Reply:
+1. Also we need to update a doc

Reply:
OK, will add these parameters to the function, defaulting to `None` (where if the params are `None`, they will be taken from the ray global node), and update the docs since the function signature will change.

Reply:
Let us know when this is done!

Reply:
Done! As per the doc, i made a brief addition on the Serve Logging documentaiton, pointing to the original Log Rotation page on Ray Core.